Core/src/TrackFinding/MeasurementSelector.cpp

```diff
@@ -64,4 +65,38 @@ double MeasurementSelector::calculateChi2(
       });
 }
 
+MeasurementSelector::Cuts MeasurementSelector::getCutsByEta(
+    const MeasurementSelectorCuts& config, double eta) {
+  const double etaAbs = std::abs(eta);
+  const auto it =
+      std::lower_bound(config.etaBins.begin(), config.etaBins.end(), etaAbs);
```

Comment:
I think lower bound does a binary search, which is slower for n ~<10. Not that it matters much. But the original code seems simpler.